kratos/utilities/auxilary_global_master_slave_relation.h

```diff
@@ -0,0 +1,213 @@
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala
+//
+//
+
+#if !defined(AUXILARY_GLOBAL_MASTER_SLAVE_RELATION)
+#define AUXILARY_GLOBAL_MASTER_SLAVE_RELATION
+// System includes
+#include <vector>
+#include <unordered_set>
+#include <iostream>
+#include <assert.h>
```

Comment:
this should not be used, use KRATOS_ERROR and its siblings for this

Reply:
I realized I am not using any asserts here. I remove the include.